### PR TITLE
fixes to grammar and added a couple of clarifying notes

### PR DESCRIPTION
**Pull request by jtfrench23**

### CODE REVIEW

**Review by ianick87 — APPROVED** [review]

File: content/docs/examples/relationaldb.md [hunk 1]
```diff
@@ -21,14 +21,17 @@ Aurora Databases are designed to integrate with Serverless environments by execu
 {{<break 1>}}
 ## Creating a Relational DB
 {{<codesnippet `/source_code/relationaldb_examples/basic_db.py`>}}
-
+{{<tool_tip key="tip" summary="Database Info">}}
```
Comment (Pia007) [inline content/docs/examples/relationaldb.md:24]: try changing tool-tip key to "info".

**Review by Pia007 — CHANGES_REQUESTED** [review]
Hi Jordan, I left one comment for a change.

**Review by DanielSanchez97 — CHANGES_REQUESTED** [review]
One small note, but besides that LGTM

**Review by ianick87 — APPROVED** [review]

File: content/docs/examples/relationaldb.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ Aurora Databases are designed to integrate with Serverless environments by execu
 {{<break 1>}}
 ## Creating a Relational DB
 {{<codesnippet `/source_code/relationaldb_examples/basic_db.py`>}}
-{{<tool_tip key="tip" summary="Database Info">}}
```
Comment (Pia007) [inline content/docs/examples/relationaldb.md]: Hi Jordan, sorry if I was not clear. --- line 24....change `key="tip"`, to `key='info`.

**Review by Pia007 — APPROVED** [review]
Hi Jordan, ignore my last comment.

File: source_code/relationaldb_examples/basic_db.py [hunk 3]
```diff
@@ -1,10 +1,14 @@
 from cdev.resources.simple.relational_db import RelationalDB, db_engine
 
+db_resource = "demo_db"
+username = "username"
+password ="pasword"
+db_name = "default_table"
 
 myDB = RelationalDB(
-  "demo_db",
```
Comment (DanielSanchez97) [inline source_code/relationaldb_examples/basic_db.py:5]: It should not use variables directly, but instead use keyword arguments. https://www.w3schools.com/python/gloss_python_function_keyword_arguments.asp

```
myDB = RelationalDB(
  cdev_name="demo_db",
  username="username",
  password="password",
  database_name="default_table"
)
```

**Review by DanielSanchez97 — CHANGES_REQUESTED** [review]
See comment about switching example files to using keyword arguments. Once that is changed, this LGTM

**Pull request merged**